moto/glue/models.py

```diff
@@ -502,13 +522,16 @@ def __init__(
         self.state = "READY"
         self.max_capacity = max_capacity
         self.security_configuration = security_configuration
-        self.tags = tags
         self.notification_property = notification_property
         self.glue_version = glue_version
         self.number_of_workers = number_of_workers
         self.worker_type = worker_type
         self.created_on = datetime.utcnow()
         self.last_modified_on = datetime.utcnow()
+        self.arn = f"arn:aws:glue:us-east-1:{get_account_id()}:job/{self.name}"
+        self.backend = backend
+        self.backend.tag_resource(self.arn, tags)
+        self.tags = tags
```

Comment:
Do we still need this assignment?
As far as I can tell, this was only used in responses.py:;filter_jobs_by_tags - and that method has refactored to use `get_tags`

Reply:
No, that assignment is no longer needed. Have removed now.